scripts/import/import_clinvar_xml

```diff
@@ -766,6 +815,28 @@ sub update_variation_set{
 
 }
 
+=head2 update_variation_feature
+- update variation_feature evidence and variation_set_id
+=cut
+
+sub update_variation_feature{
+  my $vf      = shift;
+
+  my $vf_upd_sth = $dbh->prepare(qq[
+      UPDATE variation_feature
+      SET
+        variation_set_id = CONCAT_WS(",", variation_set_id, ?),
+        evidence_attribs = CONCAT_WS(",", evidence_attribs, '$pheno_evidence_id')
+      WHERE variation_feature_id = ? ] );
+  my $new_sets = "$pheno_set_id,$clinvar_set_id";
+  if ($omimVarSet{$vf->variation->dbID}) {
+    $new_sets .=",$omim_set_id";
+  }
+  $vf_upd_sth->execute( $new_sets,  $vf->dbID );
+
+}
+
```

Comment:
This will break the fix in the recently merged PR.
If this contains hte same updates as 505, it would have been better to close 505 and update this one